.github/workflows/pr-workflow.yml

```diff
@@ -1,27 +1,25 @@
-name: pr-workflow
```

Comment:
Should we add some name here?

Without the name, the `Check` section in the `Pull-Request` simply shows the relative path to the file
i.e. `.github/workflows/pr-workflow.yml / build-us-west-2 (pull_request)`

Reply:
Yes, good catch. I didn't intend to delete the name.